mssql-plugin/src/e2e-test/features/mssql/mssql sink/RunTimeWithMacros.feature

```diff
@@ -0,0 +1,194 @@
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Source"
+    When Select plugin: "BigQuery" from the plugins list as: "Source"
+    And Navigate to the properties page of plugin: "BigQuery"
+    And Enter input plugin property: "referenceName" with value: "Reference"
+    And Replace input plugin property: "project" with value: "project.id"
+    And Enter input plugin property: "datasetProject" with value: "datasetprojectId"
+    And Enter input plugin property: "dataset" with value: "dataset"
+    And Enter input plugin property: "table" with value: "bqSourceTable"
+    Then Validate "BigQuery" plugin properties
+    And Close the Plugin Properties page
+    When Expand Plugin group in the LHS plugins list: "Sink"
+    When Select plugin: "SQL Server" from the plugins list as: "Sink"
+    Then Connect plugins: "BigQuery" and "SQL Server" to establish connection
+    Then Navigate to the properties page of plugin: "SQL Server"
+    Then Select dropdown plugin property: "select-jdbcPluginName" with option value: "driverName"
+    Then Replace input plugin property: "host" with value: "host" for Credentials and Authorization related fields
+    Then Replace input plugin property: "port" with value: "port" for Credentials and Authorization related fields
+    Then Replace input plugin property: "user" with value: "username" for Credentials and Authorization related fields
+    Then Replace input plugin property: "password" with value: "password" for Credentials and Authorization related fields
+    Then Enter input plugin property: "referenceName" with value: "sourceRef"
+    And Click on the Macro button of Property: "database" and set the value to: "DatabaseName"
+    And Click on the Macro button of Property: "fetchSize" and set the value to: "fetchSize"
+    Then Enter textarea plugin property: "importQuery" with value: "selectQuery"
+    Then Validate "SQL Server" plugin properties
+    Then Close the Plugin Properties page
+    Then Save the pipeline
+    Then Preview and run the pipeline
+    Then Enter runtime argument value "databaseName" for key "DatabaseName"
+    Then Enter runtime argument value "fetchSize" for key "fetchSize"
+    And Run the preview of pipeline with runtime arguments
+    Then Verify the preview of pipeline is "success"
+    Then Close the preview
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Enter runtime argument value "databaseName" for key "DatabaseName"
+    Then Enter runtime argument value "fetchSize" for key "fetchSize"
+    And Run the Pipeline in Runtime with runtime arguments
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+
+  @BQ_SOURCE_TEST @MSSQL_TEST_TABLE @Mssql_Required
+  Scenario: Verify pipeline failure message in logs when user provides invalid Table Name of plugin with Macros
```

Comment:
This Step definition doesn't look relatable we are simply checking if pipeline is failing with invalid macro properties and not the failure message.

Reply:
resolved